src/pages/workspaces/workspace/analysis/file-utils.ts

```diff
@@ -47,6 +55,30 @@ export interface AnalysisFileStore {
   loading: boolean
 }
 
+export const useAnalysisFiles = (): AnalysisFileStore => {
+  const signal = useCancellation()
+  const [loading, setLoading] = useState(false)
+  const workspace: WorkspaceWrapper = useStore(workspaceStore)
+  const analyses: AnalysisFile[] = useStore(analysisFilesStore)
+  const refresh = _.flow(
+    // @ts-ignore
```

Comment:
Since these issues are caused by missing types for JS modules, prefer `@ts-expect-error` over `@ts-ignore` so we can remove these comments when we do add types for them.